redisinsight/desktop/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "redisinsight",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "dev": "cross-env ELECTRON_DEV=true ELECTRON_ENABLE_LOGGING=true ELECTRON_DEBUG_LOGGING=true ELECTRON_ENABLE_STACK_DUMPING=true yarn build:preload && yarn build:renderer && yarn build && electron . --enable-logging --inspect=5858",
+    "build": "vite build --config vite.main.config.ts",
+    "build:preload": "vite build --config vite.preload.config.ts",
+    "build:renderer": "vite build --config vite.renderer.config.ts"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-react": "^4.3.3",
```

Comment:
We should not have duplicates of dependendcies
Please remove all dependencies from this file. They should be taken from root package.json. 
Or move dependencies from root to here.

Reply:
cleared the entire dependencies for the desktop. Tested to make sure it will still work.